Bestiary/Tales From the Yawining Portal Bestiary.xml

```diff
@@ -3,7 +3,7 @@
     <monster>
 		<name>Animated Table</name>
 		<size>L</size>
-		<type>construct</type>
+		<type>construct, Tales from the Yawning Portal</type>
```

Comment:
Although I agree that Tales from the Yawning Portal should be set in proper case, it is inconsistent with most of the rest of the project. 

https://github.com/ceryliae/DnDAppFiles/blob/master/Bestiary/Storm%20King's%20Thunder.xml
https://github.com/ceryliae/DnDAppFiles/blob/master/Bestiary/Monster%20Manual%20Bestiary.xml